src/content/research/agent-web-and-its-interface.md

```diff
@@ -103,6 +104,8 @@ For example, you could imagine that an agent helping to book a hotel could ask a
 
 The web has long been challenged by the on-demand nature of serving content and updates to sites - meaning you have to actually visit the site and continue doing so to get the latest information. Websockets and server-sent events helped with this but the long standing connectivity is a burden on both client and server. One of the more viable approaches to mitigating this challenge on the web came up in the world of progressive web apps (PWA). This architecture pattern leverages a file called a service worker that registers events and can be ran in isolation of the website itself allowing any client to run the JavaScript to perform the actions including prefeching, background syncing data, verifying latest information, etc. Taking a page from this architecture pattern, we can imagine how we can solve the problems of offline syncing of context, event driven actions that allow the web app to provide additional context, tool definitions, etc., and a more direct hook into dynamic user flows that they can ensure are compatible with their APIs.
 
+---
```

Comment:
Wondered about the `h1` below / article heading hierarchy; added the `hr` as a quick fix to improve the _visual_ hierarchy.